webview/src/utilities/get-workflow-tasks.ts

```diff
@@ -76,9 +75,16 @@ function categorizeEvent(eventType: EventType | undefined): Category {
     // case EventType.EVENT_TYPE_WORKFLOW_UPDATE_COMPLETED:
     // case EventType.EVENT_TYPE_WORKFLOW_PROPERTIES_MODIFIED_EXTERNALLY:
     // case EventType.EVENT_TYPE_ACTIVITY_PROPERTIES_MODIFIED_EXTERNALLY:
+    default:
+      return unhandledEventType(eventType)
```

Comment:
> We lose the compile time assertions for a runtime log with this change.

@bergundy can you clarify what you mean by this? Maybe with an example of how you are checking for compile time errors?

Reply:
The function says it returns an `EventType` so when a new event type is added this code will not compile because it will not cover the new cases and return undefined instead of EventType.

Reply:
> when a new event type is added this code will not compile because it will not cover the new cases

But this is still happening, we would just be using `never` to check for new cases. If a case is missing Typescript will still fail to compile.

We could perhaps make this even more evident by throwing an error instead ⬇️ 
```js
function unhandledEventType(eventType: never): never {
  throw new Error("Uncategorized event type: " + eventType)
}
```

Reply:
Okay, I missed that the param type is `never`, nice trick, thanks!